woocommerce/woocommerce-ios
# [Mobile Payments] Add display of reader software version to connected view

Closes #4990 

To test:
- Connect a reader and observe the connected software version. Test in landscape, portrait, light and dark modes.
- Ensure all unit tests continue to pass, and newly added tests pass as well

Changes:
- Added label for software version. Also added a missing unit test for battery level.

Screenshots:

<img src="https://user-images.githubusercontent.com/1595739/133349755-1f55c447-07b0-4bf1-89b1-505b2655ddae.PNG" width=70% />

<img src="https://user-images.githubusercontent.com/1595739/133349757-734e82dc-e33d-4659-8ef2-515e9c65f3c3.PNG" width=50% />

<img src="https://user-images.githubusercontent.com/1595739/133349762-512a6c55-fa10-4ceb-92eb-2284c7353830.PNG" width=70% />

<img src="https://user-images.githubusercontent.com/1595739/133349763-32a9866b-84b8-417f-945a-a949cf4e85ca.PNG" width=50% />

Update release notes:
